Add storage permissions to options page example (#1957)
GoogleChrome/developer.chrome.com: site/en/docs/extensions/mv3/options/index.md

    save_options);
```

> For this to work, you must add in the `manifest.json`:

Reviewer: ```suggestion
Finally, add the `"storage"` permission to the extension's `manifest.json`:
```

> ```json/3

Reviewer: ```suggestion
```json/4
```

Highlight the "storage" permission in the code block.
